css-view-transitions-1/Overview.bs

```diff
@@ -430,6 +467,12 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		: <dfn>transition suppressing rendering</dfn>
 		:: a boolean. Initially false.
+
+		: <dfn>view transition style sheet</dfn>
+		:: a [=style sheet=].
+			Initially a new [=style sheet=] in the [=user-agent origin=], ordered after the [=HTML user agent style sheet=].
+
+			Note: This is used to hold dynamic styles relating to transitions.
```

Comment:
This stylesheet is for dynamic styles involved in the transition. It's explicitly linked to a single document, clarifying the scoping.